gui/buttons.py

```diff
@@ -0,0 +1,11 @@
+import tkinter as tk
+import constants as CONST
+
+class buttonGUI(tk.Frame):
+    def __init__(self, parent):
+        buttonFrame = tk.Frame(master = parent, bg = "yellow", width = CONST.buttonsWindow.width, height = CONST.buttonsWindow.height)
+        #text = tk.StringVar()
+        #label = tk.Label(buttonFrame, textvariable = text)
+        #text.set("Buttons Part")
+        #label.pack(anchor = tk.CENTER)
```

Comment:
I left these commented lines like that. They are supposed to show a label to clarify what the frame is supposed to do. But when I run it, the background colour just disappears. I couldn't figure it out and let me know if it works in your computer.